Sources/Hub/HubApi.swift

```diff
@@ -349,7 +350,7 @@ public extension HubApi {
                 url = url.appending(component: repo.type.rawValue)
             }
             url = url.appending(path: repo.id)
-            url = url.appending(path: "resolve/main") // TODO: revisions
+            url = url.appending(path: "resolve/\(revision)")
```

Comment:
Maybe best not to do direct string interpolation?
Also needs a null check I think

Reply:
String interpolation is the preferred method for this in Swift because it's faster and benefits from compile-time optimizations. We also don't need a null check here since revision is not marked as `Optional` and always has a value. However, I can add something like `assert(revision.isEmpty == false, "Revision cannot be empty")` if you'd prefer 🙂